modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -925,6 +929,7 @@ private ObjectDefinition _addObjectDefinition(
 			counterLocalService.increment());
 
 		objectDefinition.setExternalReferenceCode(externalReferenceCode);
+
```

Comment:
I'm not sure if we need this empty line. Do you have an example in Objects module?

Reply:
I followed this example: [ObjectActionLocalServiceImpl.java#L129-L134](https://github.com/liferay/liferay-portal/blob/5930ea2f3dd66404ea6994afdeb10191522de07b/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java#L129-L134)